### PR TITLE
OPS-15474 Automate APM alerts creation

### PR DESCRIPTION
**Pull request by csalagean-ellation**
# Ticket
[OPS-15474](https://ellation.atlassian.net/browse/OPS-15474)

# Details
Automate APM/Browser alert conditions using the definition from ef_site_config.yml

Should be merged after ef_site_config.yml is merged into master.

### CODE REVIEW

**Comment by vfangellation** [comment]
I've reviewed but I would hold on merging this due to questions about the how errors would be handled and how it would be override by developers.

File: efopen/newrelic_executor.py [hunk 1]
```diff
@@ -219,6 +222,21 @@ def update_application_services_policies(self):
...
+            applications = self.newrelic.get_applications(application_name=policy.name)
+
+            if not len(applications):
+              logger.info('No applications hosted for this policy. Skip creating any APM alert')
+              continue
+
+            condition_value['entities'] = [applications[0]['id']]
+            self.newrelic.create_alert_apm_condition(policy.id, condition_value)
+          else:
+            self.update_alert_apm_condition_if_different(condition_value, policy)
```
Comment (clau-ellation) [inline efopen/newrelic_executor.py:238]: I don't think this function is defined.

Comment (clau-ellation) [reply efopen/newrelic_executor.py:238]: Otherwise everything else looks good to me. Logic looks good and I did a quick lookup of the New Relic API calls.

Comment (csalagean-ellation) [reply efopen/newrelic_executor.py:238]: Yes, I didn't commit it.

**Comment by vfangellation** [comment]
oh you got merge conflicts

**Pull request merged**